docs/releases/5.0.txt

```diff
@@ -268,9 +268,9 @@ File Uploads
 Forms
 ~~~~~
 
-* :attr:`.ChoiceField.choices` now accepts
-  :ref:`Choices classes <field-choices-enum-types>` directly instead of
-  requiring expansion with the ``choices`` attribute.
+* :attr:`.Field.choices` now accepts mappings as well as sequences, and
```

Comment:
I would use `iterable` to be consistent with the check messages and updated docs:
```suggestion
* :attr:`.Field.choices` now accepts mappings as well as iterables, and
```